src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,49 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types';
+import { useState, useEffect } from 'react';
+import { getPeople } from '../api';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
+      .then(data => setPeople(data))
+      .catch(() => setError(true))
```

Comment:
Consider setting `setError(true)` before `setIsLoading(false)` in the `catch` block to ensure that the error state is updated before the loading state is set to false. This will prevent any potential race conditions where the component might render before the error state is updated.